<launch>
    <node name="odom_estimation" type="odom_estimation_node" pkg="warpsense" output="screen"/>
</launch>